config/options.go

```diff
@@ -41,6 +43,7 @@ var (
 
     // Server
     WEB_PORT = MustNewIntOption("web.port", 8080, "The port for the web interface to serve on.");
+    WEB_MAX_FILE_SIZE = MustNewIntOption("web.maxsize", MAX_FILE_SIZE_KB, "The maximum allowed file size submitted via POST request.");
```

Comment:
Put "KB" / "kb" into both the variable name and option key.
(So when people use it they already know the units.
You can see this pattern in this same file with `TASK_MIN_REST_SECS`.)